core/services/relay/evm/evmtesting/chain_reader_interface_tester.go

```diff
@@ -231,11 +231,18 @@ func (it *EVMChainReaderInterfaceTester[T]) GetChainReader(t T) clcommontypes.Co
 
 	cr, err := evm.NewChainReaderService(ctx, lggr, lp, ht, it.client, conf)
 	require.NoError(t, err)
-	require.NoError(t, cr.Start(ctx))
 	it.cr = cr
 	return cr
 }
 
+func (it *EVMChainReaderInterfaceTester[T]) Start(t T) {
+	require.NoError(t, it.cr.Start(it.Helper.Context(t)))
+}
+
+func (it *EVMChainReaderInterfaceTester[T]) Close(t T) {
+	require.NoError(t, it.cr.Close())
+}
```

Comment:
I'm kind of confused why this needs to be in the tester interface. We already have a `Setup` function which should start the `ChainReader` and close it upon cleanup

Reply:
Thanks for this! Took a different approach, so we only expose `Close()`. 

The `Setup` function will start and stop the service in all the cases as you mention. We will use `Close()` only for cases where we want to verify that calling a method when the service is not in `Started` state returns an error